get-started/go-live.md

```diff
@@ -35,11 +40,21 @@ Most agencies add these two [snippets of code]({{ site.baseurl }}/manual/code.ht
 
 <i class="icon-check"></i> **7. Does your search box point to Search.gov?** Update your search box form code point to our service so that, when users execute a search, they'll see your Search.gov-hosted results page.
 
-	<form accept-charset="UTF-8" action="https://search.usa.gov/search" id="search_form" method="get"><div style="margin:0;padding:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" /></div>
-		 <input id="affiliate" name="affiliate" type="hidden" value="YourSiteHandle" />
-	 	<label for="query">Enter Search Term(s):</label>
-	 	<input autocomplete="off" class="usagov-search-autocomplete" id="query" name="query" type="text" />
-	 	<input name="commit" type="submit" value="Search" />
+	<form accept-charset="UTF-8" 
```

Comment:
Need to make sure this displays as a block of code, and not as an actual search box.

Reply:
Confirmed it's displaying as a code block since it's indented!